operations/mimir-mixin/dashboards/remote-ruler-reads.libsonnet

```diff
@@ -187,6 +187,25 @@ local filename = 'mimir-remote-ruler-reads.json';
     )
     .addRowIf(
       $._config.autoscaling.ruler_querier.enabled,
-      $.cpuAndMemoryBasedAutoScalingRow('Ruler-Querier'),
+      $.row('Ruler-querier - autoscaling')
+      .addPanel(
+        $.autoScalingActualReplicas('ruler_querier')
+      )
+      .addPanel(
+        $.autoScalingFailuresPanel('ruler_querier')
+      )
+    )
+    .addRowIf(
```

Comment:
Note to reviewers: I've split it into 2 rows because 5 panels in a row were too crowded even on my pretty large screen.